templates/security/certifications.html

```diff
@@ -3,202 +3,202 @@
-      </p>
-      <p>
-        Ubuntu 18.04 LTS and 16.04 LTS have both been evaluated to assurance level EAL2 through <a href="http://www.fmv.se/en/Our-activities/CSEC---The-Swedish-Certification-Body-for-IT-Security/" class="p-link--external">CSEC – The Swedish Certification Body for IT Security</a>. The consulting and evaluation testing was performed by atsec Information Security. The CSEC certification reports for <a href="https://www.commoncriteriaportal.org/files/epfiles/Certification%20Report%20Ubuntu%20LTS%2016.04.4.pdf" class="p-link--external">Ubuntu 16.04 LTS</a> and <a href="https://www.commoncriteriaportal.org/files/epfiles/Certification%20Report%20-%20Canonical%20Ubuntu%20Server%2018.04%20LTS.pdf" class="p-link--external">Ubuntu 18.04 LTS</a> are available on the CSEC website for more information.  The evaluation was performed on Intel x86_64, IBM Power8 (16.04 LTS only) and IBM Z hardware platforms.
-      </p>
-    </div>
-    <div class="col-4 u-align--center u-hide--small">
-      <img src="https://assets.ubuntu.com/v1/961a1ad1-csec-logo-removebg-preview.png?w=150" width="150" alt="">
-    </div>
-  </div>
+<section class="p-strip" id="compliance-profiles">
+ <div class="u-fixed-width">
+   <h2>Ubuntu compliance profiles</h2>
+   <p class="u-sv2">The default configuration of Ubuntu LTS releases, as provided by Canonical, balances between usability, performance and security. However, non general purpose systems can be further hardened to reduce their attack surface. Reducing the attack surface is often part of the compliance with the organization’s cybersecurity framework, but is also a widely accepted security best practice. We recommend using the industry accepted benchmarks below. Click on each benchmark for more detailed information.</p>
+   <table class="p-table--compliance-profiles">
+     <thead>
+       <th class="u-hide--small"></th>
+       <th></th>
+       <th class="u-align--center">16.04</th>
+       <th class="u-align--center">18.04</th>
+       <th class="u-align--center">20.04</th>
+     </thead>
+     <tbody>
+       <tr>
+         <td class="u-hide--small">
+         {{ image (
+           url="https://assets.ubuntu.com/v1/f98af83d-cis-logo-removebg-preview.png",
+           alt="CIS logo",
+           width="50",
+           height="50",
+           hi_def=True,
+           loading="lazy"
+           ) | safe
+         }}
+         </td>
+         <td>
+           <a href="/security/cis">Center for Internet Security (CIS) certified benchmarks for Ubuntu systems</a>
+         </td>
+         <td class="u-align--center"><i class="p-icon--success">Yes: Tooling and automation</i></td>
+         <td class="u-align--center"><i class="p-icon--success">Yes: Tooling and automation</i></td>
+         <td class="u-align--center"><i class="p-icon--success">Yes: Tooling and automation</i></td>
```

Comment:
This content should not be centered.

Reply:
Is this better? I thought it looked more even with columns centered.  

![Screenshot 2021-06-15 at 08 43 25](https://user-images.githubusercontent.com/58959073/122013117-c69ec280-cdb5-11eb-8b01-38b0c569c326.png)